server/src/lib/dokku/index.ts

```diff
@@ -2,9 +2,13 @@ import { create } from './apps/create';
 import { installed } from './plugin/installed';
 import { list } from './plugin/list';
 import { logs } from './apps/logs';
+import { listVars } from './env/listVars';
+import { add } from './env/add';
 import { deleteApp } from './apps/delete';
+import { deleteEnvVar } from './env/delete';
 
 export const dokku = {
   apps: { create, logs, deleteApp },
   plugin: { installed, list },
+  env: { listVars, add, deleteEnvVar },
```

Comment:
I think we should try to map the structure to the dokku commands, in that case instead of `env` call it config

Reply:
This makes sense, should we do also for `set` and `unset` vars as well?

Reply:
would be great :)